src/if-core.js

```diff
@@ -67,7 +67,7 @@ export class IfCore {
     return this.viewSlot.add(this.view); // Promise or void
   }
 
-  _hide() {
+  _hide(cacheView = true) {
```

Comment:
I think it's easier to define a `cache` property in base class for defining caching behavior instead of passing in as an argument. As by the time an animation has finished, the `cache` value may have been changed. @EisenbergEffect @jods4 @fkleuver

Reply:
Don't we have an issue with bindable inheritance?

Reply:
No if bindable is declared on derived class. It's a requirement, not an issue really.

About the above, it's just property value/definition on the base class, so it can be used with `this.cache` to always use the latest value to determine if it should throw away the view

Reply:
Ah. Gotcha. That makes sense. I think that would be better and you'd always be sure to have the latest value.

Reply:
Not sure if this is what you were asking fore - but here you go!

Reply:
@fracz  Nice work. That's it. Now... It would be awesome if there could be follow up PR in doc about this new feature, and 2 or some more tests for the positive cases, where `view !== null` 🚀

Reply:
Good points. One of the challenges we've had is making nice improvements like this but not getting them properly documented. If we can add the tests here and then a small bit in the documentation here: https://github.com/aurelia/documentation/blob/master/current/en-us/3.%20fundamentals/5.%20cheat-sheet.md I think we'd be ready to rock and roll. I won't block merging this for the docs. If we can get the tests in that would be good to get this merged.